tests/python/pass/test_pass_annotate_collective_ops.py

```diff
@@ -15,7 +14,6 @@
 
 @with_dialect(["tvm", "cuda"])
 @pytest.mark.skipif(not raf.build.with_cuda(), reason="CUDA is not enabled")
-@pytest.mark.parametrize("use_memory_copy_ops", [True, False])
```

Comment:
why remove this?

Reply:
broadcast and reduce with single tensor as input don't support using memory copy in PassContext. This may raise error that downcast from TensorType to TupleType fails.

Reply:
so you should not change this test, but remove broadcast/reduce from this test and annotate_collective_ops pass